lib/line/bot/v2/http_client.rb

```diff
@@ -149,6 +151,14 @@ def determine_content_type(file:)
             'application/octet-stream'
           end
         end
+
+        def normalize_headers(headers:)
+          return {} if headers.nil?
+
+          headers
+            .transform_keys(&:to_s)
+            .transform_values(&:to_s)
+        end
```

Comment:
to convert Hash[Symbol or String, String] to Hash[String, String], because Net::HTTP defines it receives Hash[String, String]. (though I think current implementation works, and it's not a problem...)